code/lib/csf-plugin/package.json

```diff
@@ -47,16 +38,34 @@
   ],
   "scripts": {
     "check": "jiti ../../../scripts/check/check-package.ts",
-    "prep": "jiti ../../../scripts/prepare/bundle.ts"
+    "prep": "jiti ../../../scripts/build/build-package.ts"
   },
   "dependencies": {
-    "unplugin": "^1.3.1"
+    "unplugin": "^2.3.5"
   },
   "devDependencies": {
     "typescript": "^5.8.3"
   },
   "peerDependencies": {
-    "storybook": "workspace:^"
+    "esbuild": "*",
+    "rollup": "*",
+    "storybook": "workspace:^",
+    "vite": "*",
+    "webpack": "*"
+  },
```

Comment:
interesting! Is this because of the `unplugin` upgrade?

Reply:
It's because we no longer have the ability to add extra externals to individual packages.

The plugin has type definitions; inside, those these packages are referenced.
So this is the correct way to have those be the same version, and also to externalize them from the bundle.